runner/internal/shim/docker.go

```diff
@@ -35,8 +37,10 @@ import (
 const ImagePullTimeout time.Duration = 20 * time.Minute
 
 const (
+	LabelKeyPrefix = "ai.dstack.shim."
 	// Set to "true" on containers spawned by DockerRunner, used for identification.
-	LabelKeyIsRun  = "ai.dstack.shim.is-run"
+	LabelKeyIsTask = LabelKeyPrefix + "is-task"
```

Comment:
(nit) Consider adding an alias, otherwise containers created by the old shim version won't be deleted by the new shim version

Reply:
I don't think backward compatibility is the issue here, since:

1. The only way to get a new `shim` version is to recreate the fleet.
2. Currently, fleet instances are destroyed and created from scratch when the fleet is recreated, thus it doesn't affect cloud instances at all.
3. There is normally only one (or zero) dead container from the last run. In the case of SSH fleet instances, if that dead container is a problem, it can be removed manually, it's a one-time action.